### DISCUSSION TITLE
Stockfish 17

### DISCUSSION THREAD

**Discussion by drooooov** [lichess-org/mobile#1288] [Ideas]
Can we get the latest stockfish 17 79 mb please?

**Comment by veloce** [comment]
Too big. The plan is to not include anymore the latest stockfish but to let the user download it.

It is planned but require more work than a simple update.

**Reply by gaajjajajaja** [reply]
Or you can give an option to user to use there custom chess engine.

**Comment by veloce** [comment]
Stockfish 17 is now available in the app with an extra download.
